plugins/node/opentelemetry-instrumentation-aws-sdk/test/sns.test.ts

```diff
@@ -1,239 +0,0 @@
-      );
-      expect(publishSpans.length).toBe(1);
-
-      const publishSpan = publishSpans[0];
-      expect(publishSpan.attributes[SEMATTRS_MESSAGING_DESTINATION_KIND]).toBe(
-        MESSAGINGDESTINATIONKINDVALUES_TOPIC
-      );
-      expect(publishSpan.attributes[SEMATTRS_MESSAGING_DESTINATION]).toBe(
-        topicName
-      );
-      expect(publishSpan.attributes['messaging.destination.name']).toBe(
-        fakeARN
-      );
-      expect(publishSpan.attributes[SEMATTRS_RPC_METHOD]).toBe('Publish');
-      expect(publishSpan.attributes[SEMATTRS_MESSAGING_SYSTEM]).toBe('aws.sns');
-      expect(publishSpan.kind).toBe(SpanKind.PRODUCER);
-    });
-
-    it('phone number', async () => {
-      const sns = new AWSv2.SNS();
-      const PhoneNumber = 'my phone number';
-      await sns
-        .publish({
-          Message: 'sns message',
-          PhoneNumber,
-        })
-        .promise();
-
-      const publishSpans = getTestSpans().filter(
-        (s: ReadableSpan) => s.name === 'phone_number send'
-      );
-      expect(publishSpans.length).toBe(1);
-      const publishSpan = publishSpans[0];
-      expect(publishSpan.attributes[SEMATTRS_MESSAGING_DESTINATION]).toBe(
-        PhoneNumber
-      );
-      expect(publishSpan.attributes['messaging.destination.name']).toBe(
-        PhoneNumber
-      );
-    });
-
-    it('inject context propagation', async () => {
-      const sns = new AWSv2.SNS();
-      const hookSpy = sinon.spy(
-        (instrumentation['servicesExtensions'] as any)['services'].get('SNS'),
-        'requestPostSpanHook'
-      );
-
-      await sns
-        .publish({
-          Message: 'sns message',
-          TopicArn: fakeARN,
-        })
-        .promise();
-
-      const publishSpans = getTestSpans().filter(
-        (s: ReadableSpan) => s.name === `${topicName} send`
-      );
-      expect(publishSpans.length).toBe(1);
-      expect(
-        hookSpy.args[0][0].commandInput.MessageAttributes.traceparent
-      ).toBeDefined();
-    });
-  });
-
-  describe('createTopic', () => {
-    it('basic createTopic creates a valid span', async () => {
-      const sns = new AWSv2.SNS();
-
-      const Name = 'my new topic';
-      await sns.createTopic({ Name }).promise();
-
-      const spans = getTestSpans();
-      const createTopicSpans = spans.filter(
-        (s: ReadableSpan) => s.name === 'SNS CreateTopic'
-      );
-      expect(createTopicSpans.length).toBe(1);
-
-      const createTopicSpan = createTopicSpans[0];
-      expect(
-        createTopicSpan.attributes[SEMATTRS_MESSAGING_DESTINATION_KIND]
-      ).toBeUndefined();
-      expect(
-        createTopicSpan.attributes[SEMATTRS_MESSAGING_DESTINATION]
-      ).toBeUndefined();
-      expect(
-        createTopicSpan.attributes['messaging.destination.name']
-      ).toBeUndefined();
-      expect(createTopicSpan.kind).toBe(SpanKind.CLIENT);
-    });
-  });
-});
-
-describe('SNS - v3', () => {
```

Comment:
This looks like it tested v3 - did you mean to delete these tests? 🤔

Reply:
Thanks for the catch! It was removed by mistake.

It's reintroduced in [`b6afcc2` (#2623)](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2623/commits/b6afcc2ccea264b9d5c9221d3faadfbe4f8c664a)